app/templates/components/object-inspector.hbs

```diff
@@ -17,7 +17,7 @@
       <button
         class="send-to-console js-send-object-to-console-btn"
         title="Send to Console"
-        {{on "click" (fn this.sendObjectToConsole @model.firstObject)}}
+        {{on "click" (fn this.sendObjectToConsole @model.lastObject)}}
```

Comment:
I'm curious why this changed from first to last?

Reply:
I think it makes more sense to to send the current object to the console (the one I digged into).
But instead it sent the initial parent object.
Can we send the current object instead? :)

Reply:
I'm not sure I understand. Clicking `$E` in the current inspector gives you the correct object, right? If so, I am curious why this changed?

Reply:
So,
I select an object to see in the inspector, then dig into a property.
In the object inspector title it will show something like `MainObject \n > property`.
If i then decide that i want to further inspect it in the console, i want to click on `$E` in the title.
But If i then click on  `$E` it sends the MainObject, but expect to get the `property` value in the console.
So, i would need to go back one stack and find the property to send it to console

Reply:
@patricklx ahhhh, this is the top level one. I thought this was for the `$E` on the individual properties. This makes sense then 👍